smac/facade/smac_facade.py

```diff
@@ -93,6 +94,9 @@ def __init__(self,
         # initial random number generator
         num_run, rng = self._get_rng(rng=rng)
 
+        # reset random number generator in config space
+        scenario.cs.random = np.random.RandomState(rng.randint(MAXINT))
```

Comment:
I think you should either do `scenario.cs.seed(rng.randint(MAXINT))` or `scenario.cs.random = rng`, but not create a new random state here.